thirdparty/ignition/v3_2/schema.go

```diff
@@ -0,0 +1,246 @@
+package types
```

Comment:
ignition is vendored here https://github.com/openshift/hypershift/blob/31ec41b3aa2a670c2a8dbfbe93634de97ad1ff85/hypershift-operator/controllers/nodepool/nodepool_controller.go#L623

https://github.com/openshift/hypershift/tree/main/vendor/github.com/coreos/ignition/v2

Reply:
thx! will remove the extra one

Reply:
updated to use existing vendored package, but switch to v3.2 which is what the installer uses in 4.9+